### PR TITLE
Daniel

### PR DESCRIPTION
**Pull request by Retsnad**
Did a bunch of pointless commits, only one worth mentioning is the audio change. Should allow us to work in peace now :D

### CODE REVIEW

File: Pirate Treasure/MainScene/IntroViewController.swift [hunk 1]
```diff
@@ -20,18 +20,18 @@ class IntroViewController: UIViewController {
...
                         }
 
     @IBAction func goToMainBoard(_ sender: AnyObject) {
         
             //stop home screen music
             let stopMenuTheme = AudioMaster()
-            stopMenuTheme.stopMenuMusic()
+            stopMenuTheme.stopMenuSFX()
             //start board game music
             let playTheme = AudioMaster()
```
Comment (chrisnyoder) [inline Pirate Treasure/MainScene/IntroViewController.swift]: you make 3 different instances of AudioMaster. Why don't you just make one? Like:
// create audioMaster instance
audioMaster = AudioMaster()

// call instance methods
audioMaster.startMenuSX()
audioMaster.stopMenuSX()
audioMaster.gameStart()

Comment (Retsnad) [reply Pirate Treasure/MainScene/IntroViewController.swift]: It's very possible I just overthought what I need to include. I thought you have to assign a type to the constant you create. For example

[    let x = AudioMaster()
    //this THEN lets me call the functions I've placed in the AudioMaster class
    x.StopMusic() ]

Is this not the case?

Comment (Retsnad) [reply Pirate Treasure/MainScene/IntroViewController.swift]: Oh I see, so you're saying basically make one constant for the audio and then call all the functionality on that one constant?

Comment (chrisnyoder) [reply Pirate Treasure/MainScene/IntroViewController.swift]: Yes exactly, you can just call one instance of the AudioMaster class, and then call all your functionality/methods with that single instance. Just cleaner.

**Pull request merged**